packages/theme-chalk/src/upload.scss

```diff
@@ -136,6 +136,9 @@
       calc(#{getCssVar('upload-dragger-padding-vertical')} - 1px);
     background-color: getCssVar('color', 'primary', 'light-9');
     border: 2px dashed getCssVar('color-primary');
+    * {
+      pointer-events: none;
```

Comment:
Do you mind explaining why this work?

Reply:
because when you drog the file on the component and move mouse， the dom element on the component will emit mouseover and mouseleave event continually，so add this css it will ignored these event when you drog and move mouse